bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/RevolutionHandler.java

```diff
@@ -66,10 +67,14 @@ protected boolean internalHandleCommand(String channelId, Command command) throw
     @Override
     protected void internalPoll() throws FreeboxException {
         super.internalPoll();
-        Config config = getManager(LcdManager.class).getConfig();
-        updateChannelQuantity(DISPLAY, LCD_BRIGHTNESS, config.brightness(), PERCENT);
-        updateChannelDecimal(DISPLAY, LCD_ORIENTATION, config.orientation());
-        updateChannelOnOff(DISPLAY, LCD_FORCED, config.orientationForced());
+        if (isLinked(GROUP_DISPLAY + ChannelUID.CHANNEL_GROUP_SEPARATOR + LCD_BRIGHTNESS)
```

Comment:
Please consider this [PR](https://github.com/openhab/openhab-addons/pull/17219) where I introduced the `private boolean anyChannelLinked(String groupId, Set<String> channelSet)` that may be more optimized from my point of view.

Reply:
Good idea.
Added.